### PR TITLE
Correct all the mistakes.

### PR DESCRIPTION
**Pull request by Concatenacion**
Correct all the mistakes and configure travis.

### CODE REVIEW

**Comment by fjlopez** [comment]
You need first be in sync with this repo. Do:
```bash
$ git fetch upstream
$ git checkout master
$ git merge upstream/master
```
Read the [wiki](https://github.com/UNIZAR-30246-WebEngineering/lab1-git-race/wiki).

Fix the issues in local and then push your changes to your repository. Warn me with a mention.

**Comment by Concatenacion** [comment]
Done @fjlopez

File: README.md [hunk 1]
```diff
@@ -5,4 +5,7 @@ Please, go to the [Wiki](https://github.com/UNIZAR-30246-WebEngineering/lab1-git
 User name | NIA | Travis-CI|Score
 ----------|-----|----------|-----
 [UNIZAR-30246-WebEngineering](https://github.com/UNIZAR-30246-WebEngineering/lab1-git-race) |30246 | [![Build Status](https://travis-ci.org/UNIZAR-30246-WebEngineering/lab1-git-race.svg)](https://travis-ci.org/UNIZAR-30246-WebEngineering/lab1-git-race)
+your name | your nia | your Travis-ci status
 Dario Sanchez Salvador | 680239 | [![Build Status](https://travis-ci.org/dari1495/lab1-git-race.svg)](https://travis-ci.org/dari1495/lab1-git-race)
```
Comment (fjlopez) [inline README.md]: Remove the line above

File: README.md [hunk 2]
```diff
@@ -5,4 +5,7 @@ Please, go to the [Wiki](https://github.com/UNIZAR-30246-WebEngineering/lab1-git
 User name | NIA | Travis-CI|Score
 ----------|-----|----------|-----
 [UNIZAR-30246-WebEngineering](https://github.com/UNIZAR-30246-WebEngineering/lab1-git-race) |30246 | [![Build Status](https://travis-ci.org/UNIZAR-30246-WebEngineering/lab1-git-race.svg)](https://travis-ci.org/UNIZAR-30246-WebEngineering/lab1-git-race)
+your name | your nia | your Travis-ci status
```
Comment (fjlopez) [inline README.md]: Remove this line. It is unnecessary now.

File: description.md [hunk 3]
```diff
@@ -1 +1,20 @@
 Complete me!
+
+
+
+## Installing Gradle on Windows 10 ##
+ 1. Download the latest Gradlle distribution, in this case, version 4.2
```
Comment (MarcosCM) [inline description.md]: Gradlle --> Gradle

File: description.md [hunk 4]
```diff
@@ -1 +1,20 @@
...
+ 2. Unpack the distribution.
+	1. Create a new directory, named Gradle.
+	2. Extract the the zip content in the newly created folder. You can unpack the distribution using an archiver tool of your choice.
+3. Configure your system environment.
+	1. In File Explorer right-click on the This PC (or Computer)  icon, then click Properties -> Advanced System Settings -> Environmental Variables.
```
Comment (MarcosCM) [inline description.md]: Please, remove a whitespace after '(or Computer)'

Comment (fjlopez) [reply description.md]: @Concatenacion can you fix this and do a new pull request, please :-)

**Pull request merged**